### PR TITLE
Updated the code to remove the deprecated www and use UnityWebRequest instead

### PR DESCRIPTION
**Pull request by AzureByte**

### CODE REVIEW

**Comment by defmys** [comment]
Thanks for the pull request!
I didn't notice it before I pushed my modification. I'll check it later and revert my changes to apply yours if everything seems to be ok.

Thank you.

File: Assets/Extensions/Unity3D-Raven-CS/Unity3DRavenCS.cs [hunk 1]
```diff
@@ -256,13 +257,19 @@ public IEnumerator AsyncSend()
             }
 
             startTime = Time.time;
-            www = new WWW(m_dsn.sentryUri, requestBuffer, headers);
-            yield return www;
+            uwr = new UnityWebRequest(m_dsn.sentryUri);
```
Comment (defmys) [inline Assets/Extensions/Unity3D-Raven-CS/Unity3DRavenCS.cs:260]: `
uwr = new UnityWebRequest(m_dsn.sentryUri, UnityWebRequest.kHttpVerbPOST);
`

It has to be a POST request.

Comment (AzureByte) [reply Assets/Extensions/Unity3D-Raven-CS/Unity3DRavenCS.cs:260]: Ah I see, let me change that.

Comment (AzureByte) [reply Assets/Extensions/Unity3D-Raven-CS/Unity3DRavenCS.cs:260]: @defmys latest commit should have that. I think the headers are already being set so the rest of it should be okay.

**Pull request merged**